### PR TITLE
New Nisqually IBM

### PR DESCRIPTION
**Pull request by bjwyman**
Three new CC values for Nisqually boards.

Signed-off-by: Brandon Wyman <bjwyman@gmail.com>
Change-Id: Id053916835eb34bef8553a94906e50c1680ca1b5

### CODE REVIEW

**Comment by bjwyman** [comment]
https://gerrit.openbmc-project.xyz/c/openbmc/entity-manager/+/51493

File: configurations/Nisqually.json [hunk 1]
```diff
@@ -55,7 +55,13 @@
         "OR",
         "com.ibm.ipzvpd.VINI({'CC': [50, 69, 50, 70]})",
         "OR",
-        "com.ibm.ipzvpd.VINI({'CC': [50, 69, 52, 48]})"
+        "com.ibm.ipzvpd.VINI({'CC': [50, 69, 52, 48]})",
```
Comment (spinler) [inline configurations/Nisqually.json:58]: you make my life hard by not just telling me the hex/ascii values for these, or why they were added :O

**Comment by geissonator** [comment]
add to approvelist

**Review by spinler — APPROVED** [review]

File: configurations/Nisqually.json (hunk 1 above)
Comment (bjwyman) [reply configurations/Nisqually.json:58]: I had plans to comment on the ASCII to hex to decimal to ...

**Pull request merged**